src/utils/util/resolver-pipeline/oscal-profile-resolve-finish.xsl

```diff
@@ -43,64 +60,130 @@
     or aliasing via use-name.
     -->
 
+    <xsl:template match="*" mode="all-or-last-only">
```

Comment:
Most changes in this file are aimed at reporting when an element doesn't belong instead of silently dropping it.